src/logging/logging.service.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { faker } from '@faker-js/faker';
+import { ClsService } from 'nestjs-cls';
+import * as winston from 'winston';
+import { RequestScopedLoggingService } from './logging.service';
+
+const mockClsService = jest.mocked({
```

Comment:
Why do we use `jest.mocked` here but not on `mockLogger`? (or the other way around)

Reply:
For mockLogger it's not strictly necessary because we're not mocking the implementation of its members. If we don't use `jest.mocked` for `mockClsService` we would get the error:
```
error TS2339: Property 'mockReturnValue' does not exist on type
```

when calling:
```
mockClsService.getId.mockReturnValue(requestId);
```

We could use `jest.mocked` for both mocks even if it's not strictly necessary for the sake of consistency, no strong opinion here 🙂